### ISSUE TITLE
Does it have retry functionality ?

### ISSUE THREAD

**Issue by lokasandeep** [gmr/env-aws-params#21]

**Comment by gmr** [comment]
Yes — `aws-sdk-go-v2` (which we use as of cf9246f) has retries enabled by default (standard mode, 3 attempts, exponential backoff with jitter). You can tune via `AWS_RETRY_MODE` (`standard` or `adaptive`) and `AWS_MAX_ATTEMPTS` env vars without any code changes. See #13 for more detail.
